app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,4 @@
+class ReviewsController < ApplicationController
+  def index
```

Comment:
You're missing an end here... do you ever actually hit this controller? If it's just to enter React, you should have an empty action.